README.md

```diff
@@ -81,3 +81,11 @@ We also welcome issues submitted about problems you encounter in using Aries VCR
 ## License
 
 [Apache License Version 2.0](https://github.com/bcgov/aries-vcr/blob/main/LICENSE)
+
+### Testing
+
+```bash
+npm install
+npm run test
+```
```

Comment:
This block should go in `docs/healthcare-vc-examples/README.md` instead, since it is specific to that set of documents and not the aries-vcr in general.